tests/Integration/Http/ResourceTest.php

```diff
@@ -989,6 +993,15 @@ public function work()
         ], $results);
     }
 
+    public function testPostTooLargeException()
+    {
+        $this->expectException(PostTooLargeException::class);
+
+        $request = Mockery::mock(Request::class,['server'=>['CONTENT_LENGTH'=>'99999999999999999']]);
```

Comment:
This test won't work on 32 bit systems. Please choose a value smaller than 2147483647 to use for the content length, there.

Reply:
I used the number 2147483640 which is < than 2147483647, should work.
Unfortunately, getPostMaxSize is protected and I'm not even sure it would work in a test. The number should work now. 
@GrahamCampbell